src/pages/sponsors.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import Link from 'gatsby-link'
+import get from 'lodash/get'
+import Helmet from 'react-helmet'
+
+import logoElysian from '../assets/images/elysian-logo.png'
+
+class Generic extends React.Component {
```

Comment:
this coulllllld just be a function component, as it's not like this actually has any lifecycle hooks or anything. 
```
const Generic = (props) => ....allTheStuffInRender
```

but I might be being silly

Reply:
(and you can even remove the lodash dependency by doing so, I think)

Reply:
also should maybe rename the component to SponsorPage or something.  In case someone ever opens it up in React devtools.

Reply:
Now I feel very silly for providing real feedback, I've been actually working today, give me a break LOL

Reply:
Real feedback is super appreciated! It's open source! We must have code standards 😄

Reply:
Converted @caffodian 🥳